src/MultiphaseStaggeredHierarchyIntegrator.cpp

```diff
@@ -134,9 +134,10 @@ MultiphaseStaggeredHierarchyIntegrator::MultiphaseStaggeredHierarchyIntegrator(
                              "CONSTANT_REFINE",
                              register_for_restart)
 {
+    // We do not use d_viscous_time_stepping_type. Use d_viscous_ts_type instead (allows for BDF2).
+    d_viscous_time_stepping_type = UNKNOWN_TIME_STEPPING_TYPE;
     if (input_db->keyExists("viscous_time_stepping_type"))
-        d_viscous_time_stepping_type =
-            IBAMR::string_to_enum<TimeSteppingType>(input_db->getString("viscous_time_stepping_type"));
+        d_viscous_ts_type = string_to_enum<TimeSteppingType>(input_db->getString("viscous_time_stepping_type"));
```

Comment:
note: using Multiphase's string_to_enum template function and enum class for TimeSteppingType